wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -779,6 +789,12 @@ instance ToJSON (V1 Core.SoftwareVersion) where
                , "version" .=  toJSON svNumber
                ]
 
+instance FromJSON (V1 Core.SoftwareVersion) where
```

Comment:
Well spotted: before the introduction of the `V1` type I had to step into the landmine of not supporting some `aeson` instance due to the fact some `Core` types had a different JSON representation which was making handling it in V1 clunky.